client/.env

```
@@ -1,2 +1,3 @@
 NODE_ENV=development
-NEXT_PUBLIC_USE_KATANA=true
+NEXT_PUBLIC_USE_KATANA=true
+REACT_APP_API_URL=http://localhost:3001
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)